stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -549,6 +561,19 @@ internal func dumpCTensorHandleContent(
   }
 }
 
+/// Given an input like "foo.tf_13" (prefix if a SIL graph function like
+/// "foo.tf_13_CPU.device_partition"), extract and return `13` as the device
```

Comment:
```suggestion
/// "foo.tf_13_CPU.device_partition"), extract and return "13" as the device
```
Nit: this is because it's not returning `13` as a numeric value, but "13" as a string.